src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -6,7 +6,7 @@
 		:style="{ minWidth: '230px', maxWidth: cardWidth }"
 	>
 		<div class="tw-grow">
-			<div>
+			<div :class="{ 'loan-card-active-hover': !allSharesReserved }">
```

Comment:
`allSharesReserved` doesn't exist on this component

Reply:
TY!